src/languages/cs.js

```diff
@@ -134,15 +172,79 @@ function(hljs) {
         ]
       },
       {
+        // Type in new declaration. Ex: new MaClasse();
+        // FIXME in 'new object();' object should be considered as keyword
+        className: 'type',
+        beginKeywords: 'new', end: '[\\(<\\[\\{]', excludeEnd: true,
+      },
```

Comment:
This would lead to inconsistencies in code: the same name will be highlighted after `new` but not highlighted in other cases. The way we do it is: highlight titles of use classes at the place of declaration and not highlight when used in the code.